src/main/java/org/openstreetmap/atlas/checks/base/CheckResourceLoader.java

```diff
@@ -83,29 +96,59 @@ public CheckResourceLoader(final Configuration configuration)
         this.configuration = configuration;
     }
 
+    public Configuration getConfiguration()
+    {
+        return this.configuration;
+    }
+
     /**
-     * Loads checks that are enabled by configuration
+     * Get configuration for a specific country, overriding for country specific overrides and group
+     * specific overrides
      *
-     * @param <T>
-     *            check type
-     * @return a {@link Set} of checks
+     * @param country
+     *            country string
+     * @return {@link Configuration}
      */
-    public <T extends Check> Set<T> loadChecks()
+    public Configuration getConfigurationForCountry(final String country)
```

Comment:
Method should be more generic. Group doesn't have to be a country.

Reply:
In this case this function is only designed for countries. The group keywords are retrieved using the country string passed in here.

Reply:
I'd suggest keeping it generic and using `group` instead of `country`.

Reply:
Thats not what I use it for. The Groups are taken into account within the function. The configuration will get an override for whatever group it is a part of. As defined by the base configuration.